.github/workflows/update-ts-version-tags.yml

```diff
@@ -31,10 +31,13 @@ jobs:
     - run: pnpm config set store-dir $PNPM_CACHE_FOLD
     - run: pnpm install
     - run: pnpm build
-    - name: retag
+    - name: clone
       run: |
         git clone --depth 1 https://github.com/DefinitelyTyped/DefinitelyTyped ../DefinitelyTyped
```

Comment:
This should probably use the checkout action, though I'm not entirely certain that supports relative paths. IMO we should be trying (long term) to not depend on DT being in the parent dir of the DT-tools codebase (at least, it's a little too much cognitive overhead for me compared to passing `--path`).

Reply:
- Is there another workflow that does this? I'll go look at the other ones again.
- Would putting in `DefinitelyTyped` instead of `../DefinitelyTyped` be OK? I can easily add a `--path` option to retag.

Reply:
ok, copied the checkout code from publish-packages.yml

Reply:
It's fine to not change the cloning, no big deal, that took a code change in the publisher which I did because I just didn't remember / trust myself to check things out in the right place...